example-configs/integreat-cms.ini

```diff
@@ -52,6 +52,8 @@ ALLOWED_HOSTS =
 STATIC_ROOT = /var/www/integreat-cms/static
 # The directory for media files [optional, defaults to "media" in the application directory]
 MEDIA_ROOT = /var/www/integreat-cms/media
+# The directory for PDF files [optional, defaults to "pdf" in the application directory]
+PDF_ROOT = /var/www/integreat-cms/pdf
```

Comment:
At the moment, only public translations can be exported to PDF.
And yes, cronjob makes sense, probably we should add this to our salt config?

Reply:
Yes, we can add this to our Salt config and we should probably document this somewhere in a deployment documentation in this repo? Otherwise the server disk would fill up over time.

Reply:
Yes, good idea. What do you think about
```
0 0 * * * /usr/bin/find /var/www/integreat-cms/pdf/* -mtime +7 -delete
```
?
We probably need this for our xliff files as well - at the moment, they're also filling up the disk space and have to be manually removed :see_no_evil: